Resources/Prototypes/_Sunrise/Roles/Jobs/USSP/ert.yml

```
@@ -121,6 +177,28 @@
     - type: Loadout
       prototypes: [ ERTLeaderGearGamma ]
 
+- type: randomHumanoidSettings
+  id: ERTLeaderGammaLock
+  parent: ERTLeaderGamma
+  components:
+    - type: GhostRole
+      requirements:
+      - !type:OverallPlaytimeRequirement
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)